mobile-app/app/screens/AppNavigator/screens/Balances/BalancesScreen.tsx

```diff
@@ -150,3 +193,126 @@ function BalanceItemRow ({
     </ThemedTouchableOpacity>
   )
 }
+
+interface DFIBalanceCardProps {
+  utxo: WalletToken
+  token: WalletToken
+  onConvertPress: () => void
+  onSendPress: () => void
+  onUtxoVsTokenPress: () => void
+  isLight: boolean
+}
+
+function DFIBalanceCard (props: DFIBalanceCardProps): JSX.Element {
+  const DFIIcon = getNativeIcon('_UTXO')
+  const totalDFI = new BigNumber(props.utxo.amount).plus(new BigNumber(props.token.amount)).toFixed(8)
+
+  return (
+    <ThemedView
+      light={tailwind('bg-white border-gray-100')}
+      dark={tailwind('bg-gray-800')}
+      style={tailwind('mx-2 mt-4 rounded-lg flex-1')}
+      testID='dfi_balance_card'
+    >
+      <ImageBackground
+        source={props.isLight ? DFIBackground : DFIBackgroundDark}
+        style={tailwind('flex-1 rounded-lg overflow-hidden')}
+        resizeMode='cover'
+        resizeMethod='scale'
+      >
+        <View style={tailwind('flex-col flex-1 mx-4 mt-5 mb-4')}>
+          <View style={tailwind('flex-row pb-3 items-center')}>
+            <DFIIcon width={24} height={24} style={tailwind('mr-2')} />
+            <ThemedText style={tailwind('pr-9 text-lg font-bold')} testID='total_dfi_label'>DFI</ThemedText>
+            <ThemedText testID='total_dfi_amount'>{totalDFI} DFI</ThemedText>
+          </View>
+
+          <View style={tailwind('flex-row pb-1.5')}>
+            <ThemedText
+              light={tailwind('text-gray-500')}
+              dark={tailwind('text-gray-400')}
+              style={tailwind('pr-16 text-sm')}
+              testID='dfi_utxo_label'
+            >
+              UTXO
+            </ThemedText>
+            <ThemedText
+              light={tailwind('text-gray-500')}
+              dark={tailwind('text-gray-400')}
+              style={tailwind('text-sm')}
+              testID='dfi_utxo_amount'
+            >
+              {new BigNumber(props.utxo.amount).toFixed(8)}
+            </ThemedText>
+          </View>
+
+          <View style={tailwind('flex-row items-center flex-1')}>
+            <ThemedText
+              light={tailwind('text-gray-500')}
+              dark={tailwind('text-gray-400')}
+              style={tailwind('pr-14 text-sm')}
+              testID='dfi_token_label'
+            >
+              Token
```

Comment:
Do we have translations for this? In ZHs and ZHT

Reply:
Looking at the current translation file, we don't translate any token name, is this special case?

Reply:
ah I'm not sure